### ISSUE TITLE
terragrunt failed to read variables file

### ISSUE THREAD

**Issue by Eliasi1** [gruntwork-io/terragrunt#1259]
hi, i am new to terragrunt and encountering a problem when try to plan-all. 
I am using terraform v.0.12.28 and terragrunt v0.23.10. 
my root terragrunt.hcl:
![image](https://user-images.githubusercontent.com/58950564/88177588-ce9bc400-cc31-11ea-8809-0c9b2b1a52b1.png)

my files-tree:
![image](https://user-images.githubusercontent.com/58950564/88177691-f854eb00-cc31-11ea-89d0-c0a6339ead1f.png)

and the problem i am encountering when trying to terragrunt plan-all:
![image](https://user-images.githubusercontent.com/58950564/88177825-233f3f00-cc32-11ea-982d-384c223b7f5a.png)

i have tried to change the file location to to c:// , but it still gave this error.
if i am using terraform to deploy each folder, it works. 

i cannot find any similar problem with terragrunt over the internet. 
does anyone encountered this too?

**Comment by yorinasub17** [comment]
Does it work if you switch `/` to `\` in the path?:
```
"${get_parent_terragrunt_dir()}\\terraform.tfvars"
```

**Comment by Eliasi1** [comment]
no, unfortunately getting the same result

**Comment by wrogrammer** [comment]
Eliasi could you remove your cache directory and after that let's try again!

**Comment by Eliasi1** [comment]
i tried it. unfortunately, it results with the same responses

**Comment by botzill** [comment]
Same issue here. Any updates?

**Comment by Eliasi1** [comment]
no.. sorry, could not solve it yet. i am using terraform for now

**Comment by vallard** [comment]
Didn't work for me either, I switched to `required_var_files = ["${get_parent_terragrunt_dir()}/common.tfvars"]` instead of arguments and that worked great.

**Comment by vallard** [comment]
curiously, this only happened for me when I used `source = ...`.  When I had it in my same dir, worked fine.
